mautrix_instagram/user.py

```diff
@@ -957,7 +968,7 @@ async def _listen(self, seq_id: int, snapshot_at_ms: int, is_after_sync: bool) -
             self.log.exception("Fatal error in listener, reconnecting in 5 minutes")
             await self.send_bridge_notice(
                 "Fatal error in listener (see logs for more info)",
-                state_event=BridgeStateEvent.UNKNOWN_ERROR,
+                state_event=BridgeStateEvent.TRANSIENT_DISCONNECT,
```

Comment:
These should probably remain as unknown errors? I think all the usual proxy issues are on the `ig-connection-error-maybe-auth` side